allennlp/tests/interpret/saliency.py

```diff
@@ -0,0 +1,53 @@
+# pylint: disable=no-self-use,invalid-name
+from pytest import approx
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.models.archival import load_archive
+from allennlp.predictors import Predictor
+from allennlp.interpret.saliency_interpreters import SimpleGradient, IntegratedGradient, SmoothGradient
+
+class TestSaliencyInterpret(AllenNlpTestCase):
+    def test_saliency_interpreters(self):
```

Comment:
Same comment on layout here; the content looks fine, it just needs to be organized correctly.  I also wonder if there's a way to test the values that come out, but that's probably more trouble than it's worth.